docs/advanced/runtime/docs/other.md

```diff
@@ -33,6 +33,70 @@ Displaying JIT debug output
 
 Use the -v -v -v -v command line argument. Set the MONO_VERBOSE_METHOD env variable to display output for only one method.
 
+
+Dumping JIT IR to IGV
+---------------------
+Set `MONO_CFG_DUMP_METHOD` to specify a method to dump over network to a running instance of the [IdealGrpahVisualizer (IGV)](http://ssw.jku.at/General/Staff/TW/igv.html).
+An IGV build that is compatible with the implementation in Mono is available for [Mac](https://www.dropbox.com/s/j08qtuk7ls2q7j8/idealgraphvisualizer_mac_b18b92bc.zip?dl=0) and [Linux/Windows](https://www.dropbox.com/s/3fwsj81df81qrya/idealgraphvisualizer_others_b18b92bc.zip?dl=0) and requires at least JRE 1.7 to run.
```

Comment:
you're right, providing the builds via Dropbox is a bit sketchy :) The builds from the official page are outdated, IGV is maintained in the Graal repository (`src/share/tools/IdealGraphVisualizer`). I forked the repo and made a release on github: https://github.com/lewurm/GraalJVMCI8/releases/tag/v0.1
I'll update the PR soon.